src/app/components/MessageBanner/index.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '../react-testing-library-with-providers';
+import MessageBanner from '.';
+import kyrgyzBanner from './fixtures';
+
+const { summaries } = kyrgyzBanner;
```

Comment:
Should we also get the first summary from the fixture data and assign it to a variable, instead of using `summaries[0]` in all of the tests?

```suggestion
const { summaries } = kyrgyzBanner;
```

Reply:
Yep if it makes it more readable - done